Package.targets

```diff
@@ -33,7 +33,7 @@
       <PackageAuthor>@(VersionInfo->'%(CompanyName)')</PackageAuthor>
       <PackageImage>http://appanacdn.blob.core.windows.net/cdn/icons/aic.png</PackageImage>
       <PackageCopyright>@(VersionInfo->'%(LegalCopyright)')</PackageCopyright>
-      <PackageCommand>$(NuGetCommand) pack "$(PackageSpecFile)" -Properties "coresdkversion=$(CoreSdkVersion);version=$(PackageVersion);author=$(PackageAuthor);image=$(PackageImage);copyright=$(PackageCopyright);configuration=$(Configuration)" -BasePath "$(BinRoot)" -OutputDirectory "$(PackageOutputDir)" -Symbols</PackageCommand>
+      <PackageCommand>$(NuGetCommand) pack "$(PackageSpecFile)" -Properties "coresdkversion=$(CoreSdkVersion);version=$(PackageVersion);author=$(PackageAuthor);image=$(PackageImage);copyright=$(PackageCopyright);configuration=$(Configuration)" -BasePath "$(BinRoot)" -OutputDirectory "$(PackageOutputDir)" $(PackageAnalysisSettings) -Symbols</PackageCommand>
```

Comment:
I thought $(PackageAnalysisSettings) is no longer needed? Should we remove it or it's consumed from some other scope?